addon/components/polaris-choice-list.js

```diff
@@ -112,6 +112,15 @@ export default Component.extend({
    */
   titleHidden: false,
 
+  /**
+   * Component to render children with a choice.
+   *
+   * @type {Component}
+   * @property extra
+   * @public
+   */
+  extra: null,
```

Comment:
Is this used? It doesn't seem to match the React implementation either, can we remove it?

Reply:
it will be...that's why I'm adding it

Reply:
Why is it needed? And since it's not used yet, can we add it when it's needed - maybe in `@smile-io/ember-smile-polaris` since it's an extension to the original Polaris spec? 🤔